src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="font">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list" >
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link" >Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link" >Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              class="nav__link "
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
```

Comment:
It's better for read
```suggestion
        >
      </a>
      
      <nav class="nav">
        <ul class="nav__list" >
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link" >Samsung</a>
          </li>
          
          <li class="nav__item">
            <a href="#smartphones" class="nav__link" >Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a
              href="#laptops-computers"
              class="nav__link "
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
          </li>
          
```

Reply:
not fixed

Reply:
still not fixed